Client/src/Common/Consts/Routes.js

```diff
@@ -7,6 +7,7 @@ export const datasetUploadRoute = '/uploadDataset'
 export const researchPaperAnalysisRoute = '/researchAnalysis'
 export const aboutRoute = '/about'
 export const profileRoute = '/profile'
+export const adminReviewRoute = '/adminReview'
```

Comment:
Later we might want to change this to datasetReview, adminReview sounds like we are reviewing admins XD